mixins/js/ez-tabs.js

```diff
@@ -107,23 +107,25 @@ window.eZ = window.eZ || {};
              *
              * @param {HTMLElement} panel
              * @param {HTMLElement} tabsLabel
+             * @param event {Event} click event
              */
-            _changeTab(panel, tabsLabel) {
-                this._unselectTab();
+            _changeTab(panel, tabsLabel, event) {
+                this._unselectTab(event);
 
                 tabsLabel.classList.add(TAB_IS_SELECTED);
                 panel.classList.add(TAB_IS_SELECTED);
             }
 
             /**
              * Unselects currently selected tabs
+             *
+             * @param event {Event} click event
              */
-            _unselectTab() {
-                const forEach = Array.prototype.forEach;
+            _unselectTab(event) {
+                const tabsContainer = event.target.closest('.ez-tabs');
+                const selectedTabs = tabsContainer.querySelectorAll('.' + TAB_IS_SELECTED);
 
-                forEach.call(this.querySelectorAll('.' + TAB_IS_SELECTED), function (element) {
-                    element.classList.remove(TAB_IS_SELECTED);
-                });
+                [].forEach.call(selectedTabs, (element) => element.classList.remove(TAB_IS_SELECTED));
```

Comment:
however I don't find the line here particular easy to read, even Array.prototype would have been cleaner then the empty array. But thats a clear imo, and if this is more the style things are done now I can perhaps get used to it.